src/components/TaskCardButtonsContainer/TaskCardButtonsContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from "prop-types";
+import { getDaysToCompleteTask } from "src/utils/compareDate";
+import DeleteButton from "src/components/Buttons/DeleteButton/DeleteButton";
+import DoneButton from "src/components/Buttons/DoneButton/DoneButton";
+
+function TaskCardButtonsContainer({
+  isTaskDone,
+  createdAt,
+  taskDoneAt,
+  handleDeleteTask,
+  handleTaskDone,
```

Comment:
onClick props should be named as prefix `on`
```suggestion
  onDelete,
  onTaskDone,
```